### PR TITLE
7.2 Incorect task/sprint/team name validation

### PR DESCRIPTION
**Pull request by SherbaKirill**
image
image
image

### CODE REVIEW

**Review by sub13 — APPROVED** [review]

File: Tests/Teams.Business.Tests/ManageTasksServiceTest.cs [hunk 1]
```diff
@@ -180,6 +180,58 @@ public async System.Threading.Tasks.Task EditTaskAsync_ManageSpiritsServiceRetur
...
+            Assert.IsTrue(result2);
+            Assert.IsTrue(result3);
+            Assert.IsTrue(result4);
+        }
+
+        [Test]
+        public async System.Threading.Tasks.Task EditTaskAsync_ManageSpiritsServiceNameCheckReturns_false()
+        {
+            //Arrange
+            var task1 = new Task { Id = 1, TeamId = 1, Name = "Ta  sk", MemberId = 1, Link = "vk.com", StoryPoints = 3, SprintId = 3 };
```
Comment (SergeyMokin) [inline Tests/Teams.Business.Tests/ManageTasksServiceTest.cs:213]: please read about TestCase

**Review by SergeyMokin — CHANGES_REQUESTED** [review]
TestCase

**Review by desenzo — APPROVED** [review]

**Review by sub13 — APPROVED** [review]

File: Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ public async Task AddSprintsAsync_ManageSpiritsServiceReturns_True()
         public async Task AddSprintsAsync_ManageSpiritsServiceReturns_False()
         {
             //Arrange
-            var sprint1 = new Sprint { Id = 1, TeamId = 1, Name = "Spr int1", DaysInSprint = 14, StoryPointInHours = 4, Status = PossibleStatuses.CompletedStatus };
+            var sprint1 = new Sprint { Id = 1, TeamId = 1, Name = "Spr  int1", DaysInSprint = 14, StoryPointInHours = 4, Status = PossibleStatuses.CompletedStatus };
```
Comment (SergeyMokin) [inline Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs:126]: ?

File: Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs [hunk 3]
```diff
@@ -140,6 +140,50 @@ public async Task AddSprintsAsync_ManageSpiritsServiceReturns_False()
             Assert.IsFalse(result3);
         }
 
+        [Test]
+        public async Task AddSprintsAsync_ManageSpiritsServiceNameCheckReturns_False()
+        {
+            //Arrange
+            var sprint1 = new Sprint { Id = 1, TeamId = 1, Name = "Spr  int1", DaysInSprint = 14, StoryPointInHours = 4, Status = PossibleStatuses.CompletedStatus };
```
Comment (SergeyMokin) [inline Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs:147]: why 2 spaces?

File: Tests/Teams.Business.Tests/ManageTasksServiceTest.cs [hunk 4]
```diff
@@ -161,7 +161,7 @@ public async System.Threading.Tasks.Task EditTaskAsync_ManageTaskServiceReturns_
         public async System.Threading.Tasks.Task EditTaskAsync_ManageSpiritsServiceReturns_False()
         {
             //Arrange
-            var task1 = new Task { Id = 1, TeamId = 1, Name = "Ta sk", MemberId = 1, Link = "vk.com", StoryPoints = 3, SprintId = 3 };
+            var task1 = new Task { Id = 1, TeamId = 1, Name = "Ta  sk", MemberId = 1, Link = "vk.com", StoryPoints = 3, SprintId = 3 };
```
Comment (SergeyMokin) [inline Tests/Teams.Business.Tests/ManageTasksServiceTest.cs:164]: same

**Review by SergeyMokin — CHANGES_REQUESTED** [review]
Please, when you add two spaces one after another remove duplicated spaces (in names)

File: Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs (hunk 3 above)
Comment (SherbaKirill) [reply Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs:147]: I made it impossible to put several spaces in a row and displayed it here

Comment (SergeyMokin) [reply Tests/Teams.Business.Tests/ManageSprintsServiceTest.cs:147]: alright then

**Review by SergeyMokin — APPROVED** [review]

**Pull request merged**